kaggle/pipeline.go

```diff
@@ -94,3 +84,9 @@ func main() {
 		gravyutil.FatalIfErr(err)
 	}
 }
+
+func ParseOrDie(floatString string) float64 {
```

Comment:
While we're at it, this should go wherever the utilities are (and be called "StringToFloatOrDie").

(BTW, I've kinda totally rejected the "handle every error" philosophy of Go with all these "OrDie" functions. We might want to revisit that decision if we move to using channels / become asynchronous.)

Reply:
Ah cool. A common pattern that's used in the go standard library for these types of things is to prefix the function with `Must`

https://golang.org/pkg/regexp/#MustCompile